flink-cdc-runtime/src/test/java/org/apache/flink/cdc/runtime/serializer/event/EventSerializerTest.java

```diff
@@ -44,7 +45,12 @@ protected Class<Event> getTypeClass() {
 
     @Override
     protected Event[] getTestData() {
-        Event[] flushEvents = new Event[] {new FlushEvent(1), new FlushEvent(2), new FlushEvent(3)};
+        Event[] flushEvents =
+                new Event[] {
+                    new FlushEvent(1, SchemaChangeEventType.CREATE_TABLE),
+                    new FlushEvent(2, SchemaChangeEventType.CREATE_TABLE),
+                    new FlushEvent(3, SchemaChangeEventType.CREATE_TABLE)
+                };
```

Comment:
Please also test if FlushEvents with extra TableIds could be correctly [de]serialized.